modules/doc/content/newsletter/2022_01.md

```diff
@@ -0,0 +1,9 @@
+# MOOSE Newsletter (January 2022)
+
+## MOOSE Improvements
+
+## Bug Fixes and Minor Enhancements
+
+- An option was added to [PiecewiseLinear.md] to allow extrapolation to occur
+  when the sample is outside of the provided abscissa values, instead of using
+  the nearest ordinate value. This is accomplished with a new parameter `extrap`.
```

Comment:
Could you make clear to the reader that this parameter is a boolean, and that `extrap = true` is required to allow extrapolation? I know this is in the doco, but I don't think it hurts to say here.